content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -226,9 +226,22 @@ kubectl describe pvc <pvc_name>
 ```
 
 If the `PersistentVolumeClaim` has the status `FileSystemResizePending`, it is safe to recreate the pod using the PersistentVolumeClaim.
+{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
+FlexVolumes allow resize if the driver is set with the `RequiresFSResize` capability to true. Since this is a alpha feature
+both  `ExpandInUsePersistentVolumes` and `ExpandPersistentVolumes` feature gates must be enabled.
```

Comment:
What is alpha feature? Flex volume resizing is not feature gated. Flex volume does not need `ExpandInUsePersistentVolumes` for flex volume resizing. Lets not conflate these two. Also `ExpandPersistentVolumes` is already in beta and hence enabled by default.